src/dmd/globals.h

```diff
@@ -64,6 +64,12 @@ enum CPU
     native              // the machine the compiler is being run on
 };
 
+enum CppStdRevision
+{
+    CppStdRevisionCpp98,
+    CppStdRevisionCpp11,
```

Comment:
Don't forget to use the same values here as well ;)

Reply:
Fixed.